app/javascript/packs/banner_editor.jsx

```diff
@@ -294,16 +294,16 @@ export default class BannerEditor extends React.Component {
   addLink() {
     switch (this.state.linkOption) {
       case "Youtube":
-        this.setState({ youtube: this.state.currentLink });
+        this.setState({ youtube: "https://www.youtube.com/" + this.state.currentUsername });
```

Comment:
Most channels follow like this `channel/UCrpFW99zAUaJT8cKmqeagkg` or `user/username` so we might want to make it similar to that

Reply:
@corymcdonald it gets redirected to `user/username`. All youtube channels can create their own username, which seems more visually friendly by having usernames rather than channel id. This change won't break back-compat, so existing site channels will be okay.